setup.py

```diff
@@ -23,6 +23,7 @@
 runtime = set([
     'pyyaml>=3.10,<=3.13',
     'six',
+    'requests_cache',
```

Comment:
Since this will pull in `requests` and we have a specific version requirement in the `develop` target, you should probably move that into `runtime`.

Reply:
Moved the 'requests==2.13.0' def to runtime